core/controllers/feature_flag.py

```diff
@@ -62,6 +62,6 @@ def get(self) -> None:
         self.render_json({
             'msg': 'ok',
             'is_enabled': feature_flag_services.is_feature_flag_enabled(
-                self.user_id, feature_flag_list.FeatureNames.
-                DUMMY_FEATURE_FLAG_FOR_E2E_TESTS.value)
+                feature_flag_list.FeatureNames.DUMMY_FEATURE_FLAG_FOR_E2E_TESTS
+                .value, user_id=self.user_id)
```

Comment:
Maybe break after the comma, since .value is part of the previous argument.

Perhaps better: create a variable to store the first argument above the render_json call, then reference it here.

Reply:
Done. I went with the latter.